packages/asset/contracts/Asset.sol

```diff
@@ -65,7 +71,10 @@ contract Asset is
     /// @notice Mint new token with catalyst tier chosen by the creator
     /// @dev Only callable by the minter role
     /// @param assetData The address of the creator
-    function mint(AssetData calldata assetData) external onlyRole(MINTER_ROLE) {
+    function mint(
+        AssetData calldata assetData,
+        string memory assetUri
```

Comment:
I think the name of the variable is confusing, the value passed here will not be an URI it will be the ipfsHash right?